src/interrex_to_movebank.Rmd

```diff
@@ -70,14 +70,8 @@ movebank_ref_data <-
       as.POSIXct(track_session_end_date, tz = "UTC", format = "%Y-%m-%d %H:%M:%S"),
       "%Y-%m-%dT%H:%M:%SZ"
     ),
-    `deploy-on-latitude` = dplyr::case_when(
```

Comment:
I note that the coordinates for `Ijzermonding slikken` have been added to the spreadsheet, so they are not lost 👍